"operators_mo_mtx_elements" doesn't run when mpi is enabled (even if test doesn't use mpi)
see: https://ccportal.ims.ac.jp/en/node/2411

Author: Pavel Grochal (INUITS)
Comments from B. Hajgato (UGent) at OCT 5th 2020:
In the test python program both ignore_sign and ignore_order is set to True,
but ignore_order does not seem to work perfectly.
Using ifort 2020.1 and MKL2020.1 on cascadelake a manual comparison passed, meanwhile the automatic
comparison failed. Therefore, test will be disabled. 
diff -ru DIRAC-19.0-Source.orig/cmake/custom/test.cmake DIRAC-19.0-Source/cmake/custom/test.cmake
--- DIRAC-19.0-Source.orig/cmake/custom/test.cmake	2019-12-12 17:26:15.000000000 +0100
+++ DIRAC-19.0-Source/cmake/custom/test.cmake	2020-09-24 12:54:02.524830820 +0200
@@ -158,9 +158,8 @@
 dirac_test(lucita_q_corrections "short;qcorr;ci" "")
 dirac_test(x2c-SCF_to_4c-SCF "short;x2c;4c;scf" "")
 dirac_test(import_mos "short;import;scf;mcscf" "")
-dirac_test(operators_mo_mtx_elements "short;operators" "")
+# dirac_test(operators_mo_mtx_elements "short;operators" "")
 dirac_test(spinorbit-from-spinfree "short;scf;resolve" "")
 dirac_test(spinrot "short;response" "")
 # disabing eomcc expectation value test until the code is verified
 #dirac_test(eomcc_expval "short;cc" "")
-

